### PR TITLE
Fixes #317: Add redux-airbrake to examples

### LINKED ISSUE #317: Add redux-airbrake to examples
Hello,

I created a Redux middleware for airbrake-js. You can check it out here:
https://github.com/alexcastillo/redux-airbrake

Maybe it would be worth adding it as an example in this repo?
I would be happy to submit a PR.

### PR DESCRIPTION
**Pull request by alexcastillo**

### CODE REVIEW

File: examples/redux/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+``` js
+import airbrakeMiddleware from 'redux-airbrake';
+```
+
+#### 3. Configure & add middleware
+``` js
+const errorMiddleware = airbrakeMiddleware({
+    projectId: '******',
+    projectKey: '**************'
+});
```
Comment (vmihailenco) [inline examples/redux/README.md]: Could you change this to accept airbrake-js Client? This way
- it is consistent with other middlewares
- user can configure client by adding filters/additional info

Comment (alexcastillo) [inline examples/redux/README.md]: The middleware is actually using airbrake-js client behind the scenes. See its implementation here: https://github.com/alexcastillo/redux-airbrake/blob/master/src/index.js#L12

It already supports configuration for filters, etc.

Comment (vmihailenco) [inline examples/redux/README.md]: So what about accepting airbrake-js Client instance? Do you see any drawbacks?

>It already supports configuration for filters, etc.

For example, how can I add a filter?

**Comment by vmihailenco** [comment]
@alexcastillo ping. Are you still interested?

**Comment by alexcastillo** [comment]
@vmihailenco yes, I just need a little time. I'll get back to you soon.

Thanks.

**Comment by vmihailenco** [comment]
Okay, thanks. Just wanted to mention that following [code](https://github.com/alexcastillo/redux-airbrake/blob/master/src/index.js#L13-L19) is incorrect, because you add new filter with every `airbrakeLog` call. So after 100 calls you will have 100 filters. It is better to rewrite that code as:

```
airbrake.notify({error: err, params: params});
```

**Comment by alexcastillo** [comment]
Will this be merged?

**Comment by vmihailenco** [comment]
Did you fix any issues I mentioned?

**Comment by alexcastillo** [comment]
Thanks for the feedback, everyone.
I've updated the PR to reflect the changes asked.

**Comment by vmihailenco** [comment]
Thank you!

**Pull request merged**